cli/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.20.1
 	github.com/aws/aws-sdk-go-v2/service/dynamodb v1.18.1
 	github.com/aws/aws-sdk-go-v2/service/sts v1.18.2
-	github.com/chanzuckerberg/go-misc v1.0.3
+	github.com/chanzuckerberg/go-misc v1.0.4
```

Comment:
Should the dependency on `github.com/aws/aws-sdk-go` be gone now? I still see it below

Reply:
How about now?